### PR TITLE
issue81 Updating boiler-plate text for technology preview.

### PR DESCRIPTION
**Pull request by stoobie**
This PR addresses #81.

### CODE REVIEW

**Comment by stoobie** [comment]
@leswilliams44 @IngridT1 @bergerhoffer @rkratky Please review.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc [hunk 1]
```diff
@@ -101,13 +101,17 @@ Workaround: Rerun the deployment command, or use a local container image registr
 [discrete]
 ==== Technology Preview
 
-Use present tense in the following format:
+Use present tense in the following format, where `<TechPreviewName>` is the name of your product, component or feature:
+
+// The following is based on RHEL 9 snip_techpreview.adoc
 ----
-In <release>, a Technology Preview is available for <feature explanation>.
+<TechPreviewName> is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:108]: To follow our replaceable value guidance can you change `<TechPreviewName>` to `<tech_preview_name>` here and above?

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc [hunk 2]
```diff
@@ -101,13 +101,17 @@ Workaround: Rerun the deployment command, or use a local container image registr
...
 ----
+// PREVIOUS GUIDANCE
+// In <release>, a Technology Preview is available for <feature explanation>.
 .Example doc text
 ----
-In Red Hat OpenStack Platform 16.0, a Technology Preview is available for the Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants.
+In Red Hat OpenStack Platform 16.0, the Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:114]: I might consider using a more concise example here. This one is kind of long and hard to follow.

A few examples I see when searching the OpenShift docs:
* OpenShift Serverless Functions is a...
* Using SSPI connection support on Microsoft Windows is a...
* Hot-plugging virtual disks is a...
* The Driver Toolkit is a...

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc [hunk 3]
```diff
@@ -101,13 +101,17 @@ Workaround: Rerun the deployment command, or use a local container image registr
 [discrete]
 ==== Technology Preview
 
-Use present tense in the following format:
+Use present tense in the following format, where `<TechPreviewName>` is the name of your product, component or feature:
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:104]: I'd suggest to add something in this intro statement to make it clearer that writers are to use this text EXACTLY as is (just inserting the name of the feature at the beginning).

**Review by bergerhoffer — COMMENTED** [review]
Thanks @stoobie! Left a few suggestions

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc (hunk 1 above)
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:108]: Fixed.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc (hunk 2 above)
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:114]: @bergerhoffer I think that having multiple examples of <tech_preview_name> would be helpful, so I added a few and changed the boiler plate example.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc (hunk 3 above)
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: Done.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc [hunk 4]
```diff
@@ -101,13 +101,27 @@ Workaround: Rerun the deployment command, or use a local container image registr
...
+* SSPI connection support on Microsoft Windows
+* Hot-plugging virtual disks
+* The Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants
+
 .Example doc text
+
 ----
-In Red Hat OpenStack Platform 16.0, a Technology Preview is available for the Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants.
+The Driver Toolkit is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
 ----
```
Comment (dfitzmau) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:126]: How about adding an "Additional resources" section that lists the following link. Although, this link is available in the examples.

https://access.redhat.com/support/offerings/techpreview

The DPM and CS on the project I work on often point me to this source.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc [hunk 5]
```diff
@@ -101,13 +101,27 @@ Workaround: Rerun the deployment command, or use a local container image registr
 [discrete]
 ==== Technology Preview
 
-Use present tense in the following format:
+Use the following text verbatim, where `<tech_preview_name>` is the name of your product, component or feature:
```
Comment (dfitzmau) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:104]:
```suggestion
Use the following text verbatim, where `<tech_preview_name>` is the name of your product, component, or feature:
```

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc [hunk 6]
```diff
@@ -101,13 +101,27 @@ Workaround: Rerun the deployment command, or use a local container image registr
...
+
+// The following is based on RHEL 9 snip_techpreview.adoc
 ----
-In <release>, a Technology Preview is available for <feature explanation>.
+<tech_preview_name> is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
 ----
+
+// PREVIOUS GUIDANCE
```
Comment (dfitzmau) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:111]: Remove these two comments? Or do we need to keep a record of previous content?

**Review by dfitzmau — COMMENTED** [review]
Hello @stoobie . I've added some minor feedback. Please let me know your thoughts.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc [hunk 7]
```diff
@@ -101,13 +101,27 @@ Workaround: Rerun the deployment command, or use a local container image registr
...
+* The Driver Toolkit
+* SSPI connection support on Microsoft Windows
+* Hot-plugging virtual disks
+* The Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants
+
 .Example doc text
+
 ----
-In Red Hat OpenStack Platform 16.0, a Technology Preview is available for the Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants.
+The Driver Toolkit is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
```
Comment (ndeevy) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:124]: Hey @stoobie this all looks like great changes here, just a thought: would it be possible to change the anthropomorphic 'enabling' (because, why not if possible) but no worries if it needs to be in line with legal wording etc.

**Review by ndeevy — DISMISSED** [review]

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc (hunk 7 above)
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:124]: Hey @ndeevy,

Thanks for reviewing!

I took this text verbatim as noted in #81.

But this is the first time I've ever heard any issue with the word "enabling". How is it anthropomorphic? What do you use instead?

Comment (ndeevy) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:124]: Ok cool, I guess it's legal team approved then since it's from that link in #81  Apologies for the noise, we can leave it as is, there might be legal reasoning behind the words here. 

For the sake of discussing the mechanics of language, style, and grammar though ... :-D  certain words jump out that might indicate anthropomorphism. Like allow, let, permit, and enable. If the idea is that an object, technology, or non-human entity inherits human abilities or gives permission to a human to do something, it's probably anthropomorphic and puts the human in a subordinate position below the object. Usually this can be reworked to put the human in front so it's user-oriented, (sometimes not though because RH technologies do sometimes inherit human abilities).

In this case, I'd probably change it to something like:

`These features provide customers early access to upcoming product features to test functionality and provide feedback during the development process.`

There's an entry in the _IBM Style Guide: quick reference_ on confluence too if you want to look there -any feedback very welcome.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc (hunk 4 above)
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:126]: I think because it's already available in the example, adding it again under Additional resources doesn't really add any value.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc (hunk 5 above)
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:104]: Fixed.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc (hunk 6 above)
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:111]: Removed.

**Review by dfitzmau — APPROVED** [review]

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc (hunk 7 above)
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:124]: Hmmm...I do see what you mean. I never really liked "enable" so I try to avoid it, but it's nice to see it mentioned in the style guide. Thanks for pointing that out!
This particular text is not something that I'm going to change. It's been hammered out already.

**Comment by stoobie** [comment]
@dfitzmau @ndeevy I implemented your changes. Can you please approve?

**Comment by dfitzmau** [comment]
HI @stoobie . I thought I approved the pull request. Did I do this incorrectly? Please let me know of the correct steps.

**Comment by stoobie** [comment]
@dfitzmau You did it correctly, but because I made changes, you need to do it again. When changes are made, they cancel out the existing approval.

**Comment by dfitzmau** [comment]
> @dfitzmau You did it correctly, but because I made changes, you need to do it again. When changes are made, they cancel out the existing approval.

Ah, OK!. I'll approve again,

**Review by dfitzmau — APPROVED** [review]

**Review by ndeevy — APPROVED** [review]

**Comment by ndeevy** [comment]
LGTM thanks @stoobie

**Comment by stoobie** [comment]
@bergerhoffer Can you please take a look and (hopefully) approve?
